src/fluka/fluka_funcs.cpp

```diff
@@ -833,7 +834,8 @@ void fludag_write(std::string matfile, std::string lfname) {
   if (workflow_data.material_library.size() != 0) {
     // write COMPOUND CARDS
     std::ostringstream material_str;
-    fludag_all_materials(material_str, workflow_data.material_library);
+    pyne::mat_map mat_lib = workflow_data.material_library.get_mat_library();
+    fludag_all_materials(material_str, mat_lib);
```

Comment:
why not pass the material library directly as you can use it as a std::map

Reply:
Are you saying we should leave the function arguments alone and pass this the `mat_lib` variable?

Reply:
I think we should pass directly the material_library object
```suggestion
    fludag_all_materials(material_str, workflow_data.material_library);
```

Reply:
Ah, nevermind. I see what your saying. Changes pending...